1208-projects/project_wizard.feature

```diff
@@ -0,0 +1,190 @@
+Feature: User edits Project
+    In order to document a project 
+    As a logged in User
+    I want to edit existing and create new projects
+
+    @xxx
+
+    Scenario Outline: User sees the Project Wizard
+        Given User opens My Projects
+        When they click Create New Project Button
+        Then the User is the Project Owner
+        And they see tabs:
+            | Description  |
+            | Participants |
+            | Financing    |
+        And they see the Description tab
+        And they see fields:
+            | Title                    |
+            | Summary                  |
+            | Coordinating Institution |
+            | Start Date               |
+            | End Date                 |
+            | Internal Project Code    |
+        And they see a Support Button
+        And they see a Button "<Persist>" desided by Project's "<Status>"
+        And they see a Next Button
+        Examples:
+            | Status    | Persist |
+            | Published | Publish |
+            | Draft     | Save    |
```

Comment:
jeg syns det gir mest mening om det alltid er en Save-knapp der, mens Publish-knapp kommer i tillegg hvis prosjektet ikke allerede er publisert. Også litt rart at man publiserer Prosjekt på en annen måte enn hva som er tilfelle for Registration, hvor man bare kan publisere fra Landing Page.

Dette burde egentlig avklares med en UX-person vi ikke har for øyeblikket.. :/ men nå har vi i hvert fall bare en versjon av Prosjekt å forholde oss til, så det er kanskje greit nok for nå

Reply:
Godt innspill. Eg likte ikkje denne løysninga sjølv helder, men eg "måtte" avslutte i går før sola smelta meg bort. Vi har ein liknande løysning på publikasjon, med fire ulike knapper. Der bruker vi "Lagre kladd" og "Lagre og presenter" i wizarden og "Publiser" på Landingpage. Så bruker vi "Lagre" når ein publisert ressurs blir endra i wizard. Eg er ikkje overbevis om at det er ein betre løysning - men vi bør uansett ha ein felles måte å gjer det på. Så eg vil prøve å tilpasse prosjektflyten til publikasjonsflyten.

Reply:
Det er tilfellet der ein bruker editerer på til dømes ein deltakar eit publisert prosjekt og så skal lagre dette som gjer at det blir litt komplisert. Eg likar ikkje:
*  å tvinge personen innom landingpage, for å finne "publiser"-knappen
*  å endre "Lagre kladd"-knapp til "Publiser"-knapp (for prosjektet er alt publisert)

Eg går for å endre "Lagre kladd"-knapp til "Lagre"-knapp.

(Evt kan det vere at "Oppdater" eller "Oppdater publisert publikasjon/prosjekt" er ein betre formulering - men det må vere konsistent på begge plasser)

Reply:
Eg gikk for "Oppdatere"-varianten - den er mest sjølvforklarande
`        Examples:
            | Status    | Button                   |
            | Published | Update published project |
            | Draft     | Save draft               |
`

Reply:
Så flytta eg "publiser" til landingpage og la til "Lagre og presenter" på finansiering.
Fjerna lagre option på beskrivelses fanene, API forutsetter ein PM.